tests/unit/library.cairo

```diff
@@ -114,7 +117,9 @@ namespace it {
         let (local old_slot) = ERC3525.slot_of(from_token_id);
 
         %{
-            stop_prank = start_prank(ids.caller) 
+            stop_prank = start_prank(ids.caller)
+            mock_call(ids.to, "onERC3525Received", [ids.IERC3525_RECEIVER_ID])
+            mock_call(ids.to, "supportsInterface", [1])
```

Comment:
to be consistent with the other test, do we want to change `[1]` to `[ids.TRUE]`  ?